src/Endpoints/class-base-api-proxy.php

```
@@ -148,12 +148,18 @@ protected function get_data( WP_REST_Request $request, bool $require_api_secret
 		}
 
 		// A proxy with caching behavior is used here.
-		$response = $this->proxy->get_items( $params );
+		$response = $this->proxy->get_items( $params ); // @phpstan-ignore-line.
+
+		if ( false === $response ) {
+			return new stdClass();
+		}
```

This `false` case was missing and PHPStan were complaining about it so added it.

---

I wonder if this should return an error instead of an empty `stdClass`. Thoughts?

---

I think returning empty `stdClass` is better ...... because in reality it is not an error so we also shouldn't treat it like error.